deps/rust/BUILD.bazel

```diff
@@ -70,31 +73,44 @@ crates_vendor(
     ],
 )
 
-# TODO(cleanup): Switching to rust_library() would reduce the overhead of this by not creating a
-# crate-of-crates but this previously broke the Windows build.
+# define the path to the lolhtml headers, this is clunky but allows us to avoid cloning the
+# repository twice.
+alias(
```

Comment:
This alias is no longer being used based on `deps/rust/BUILD.lolhtml` – I think the approach we had so far made it easy to find and update the lolhtml version number if needed, but I'm not attached to it so leaving BUILD.lolhtml and deleting this alias is an option too